packages/next/client/image.tsx

```diff
@@ -353,6 +353,9 @@ export default function Image({
         )}`
       )
     }
+    if (Number.isNaN(widthInt) || Number.isNaN(heightInt)){
```

Comment:
`width` and `height` are not required when `layout="fill'` right? That seems like it would throw here

Reply:
You are right. it only can handling with `layout='responsive'`
should change 
```
if (layout === 'responsive' && (Number.isNaN(widthInt) || Number.isNaN(heightInt))) {
  throw new Error(`Image with src "${src}" has invalid "width" or "height" property. These should be numeric values.`);
}
```

when `layout='fill'` it doesn't need width and height. so don't know that value.
so, there is another process like below.

```
if (layout !== 'fill' && ((widthInt || 0) * (heightInt || 0) < 1600)) {
  console.warn(`Image with src "${src}" is smaller than 40x40. Consider removing the "placeholder='blur'" property to improve performance.`);
}
```

But, I don't like that, because that is not abstract enough I think.
I tried getting real width from image, but failed. 😂

Reply:
Are we sure this would have thrown on an empty image with layout fill? I believe widthInt is undefined if nothing is provided, and Number.isNaN(undefined) is false

Reply:
Yes, Number.isNaN(undefined) return false. but isNaN(undefined) return true.
It changed Number.isNaN to isNaN

Reply:
No. as i said, this PR can't handle when `layout='fill`. when `layout='fill` it doesn't need width and height. and don't know it's real size.

And, as you said Number.isNaN(undefined) return false. but It changed  to isNaN.
isNaN(undefined) return true.